site/app/templates/admin/admin_gradeable/AdminGradeableAuto.twig

```diff
@@ -4,89 +4,92 @@
             What are students allowed to do?
         </div>
         <div>
-            View the gradeable on the course home page and download submitted files?<br>
-            <input type="radio" id="no_student_view_after_grades" name="student_view_after_grades" value="false"
-                    {{ gradeable.isStudentView() and not gradeable.isStudentViewAfterGrades() ? 'checked' : '' }}/> Yes
-            <input type="radio" id="no_student_view" name="student_view" value="false"
-                    {{ not gradeable.isStudentView() ? 'checked' : '' }}/> No
-            <input type="radio" id="yes_student_view_after_grades" name="student_view_after_grades" value="true"
-                    {{ gradeable.isStudentView() and gradeable.isStudentViewAfterGrades() ? 'checked' : '' }}/> When grades are released
-
-            <input type="radio" id="yes_student_view" name="student_view" value="true"
-                    {{ gradeable.isStudentView() ? 'checked' : '' }} hidden/>
+            <fieldset>
+                <legend>View the gradeable on the course home page and download submitted files?</legend><br>
```

Comment:
The UI appears to be unaffected except space in between this legend and the associated radio buttons is added.  Can be fixed by just removing the now redundant <br>.  (Same thing exists on line 24)

Reply:
Certainly could be.  I had removed some `<br />` tags in other code for that very reason.  I'll have another look.

Reply:
Done